### ISSUE TITLE
404 No found on line 2902

### ISSUE THREAD

**Issue by antoniohof** [networked-aframe/networked-aframe#86]
Hi, 
Ive just updated to the latest version.
Great job with the ownership transfer and reconnect features!

My project is connecting fine, but this line is constantly giving 404 error:
function 'updateTimeOffset'
line 2902
`return fetch(document.location.href, { method: "HEAD", cache: "no-cache" }).then(function (res) {
`

I'm using localhost, maybe its that?

Thank you

**Comment by haydenjameslee** [comment]
It should still work on localhost. Which adapter are you using?

**Comment by antoniohof** [comment]
Easyrtc on the beta branch, for supporting ios audio

**Comment by haydenjameslee** [comment]
You can switch to 0.5.0 on npm now, it has that iOS audio fix. Not sure why easyrtc isnt working. Traveling today so won't be able to check it out. Is it a red error in the JS console? Earlier today somebody had a similar report but it was due to them having XHR requests being logged, which is a chrome dev tools option.

**Comment by antoniohof** [comment]
I found the problem.
Single page applications (SPA) use fake urls, so the window.location.href won't point to a real page, it returns 404. That was the problem. I configured the server to always redirect 404 to the index, as stated here:
https://router.vuejs.org/en/essentials/history-mode.html

Another fix would be to use window.location.origin instead of window.location.href. But I don't know the implications on your code.
